src/battle_util.c

```diff
@@ -9031,12 +9031,14 @@ static inline u32 CalcMoveBasePowerAfterModifiers(u32 move, u32 battlerAtk, u32
         modifier = uq4_12_multiply(modifier, (B_TERRAIN_TYPE_BOOST >= GEN_8 ? UQ_4_12(1.3) : UQ_4_12(1.5)));
     if (IsBattlerTerrainAffected(battlerAtk, STATUS_FIELD_PSYCHIC_TERRAIN) && moveType == TYPE_PSYCHIC)
         modifier = uq4_12_multiply(modifier, (B_TERRAIN_TYPE_BOOST >= GEN_8 ? UQ_4_12(1.3) : UQ_4_12(1.5)));
+
+    #define SPORT_DAMAGE_REDUCTION B_SPORT_DMG_REDUCTION >= GEN_5 ? 0.33 : 0.5
```

Comment:
Personally I think the define is unnecessary.

Reply:
It's removed now